releasenotes/notes/0.8-prelude-3057999fac96bf43.yaml

```diff
@@ -0,0 +1,7 @@
+---
+prelude: >
+    From this release, Qiskit Machine Learning requires Qiskit `1.0` or above, with important
```

Comment:
Should we also have this in an upgrade section too. I,e Qiskit 1.0 is now the minimum version required so if you were still using an earlier version you should upgrade. That way if a user just looks at what they need to upgrade in that section they won't miss out on that fact if they did not see the prelude.

Reply:
I added it to the update section of the Algo PR reno.